WordPressComStatsiOS/WordPressComStatsiOS/UI/WPStatsViewController.m

```diff
@@ -50,7 +50,7 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
         self.statsTableViewController = tableVC;
         tableVC.statsDelegate = self.statsDelegate;
         tableVC.statsProgressViewDelegate = self;
-        tableVC.statsService = [[WPStatsService alloc] initWithSiteId:self.siteID siteTimeZone:self.siteTimeZone oauth2Token:self.oauth2Token andCacheExpirationInterval:5 * 60];
+        tableVC.statsService = [[WPStatsService alloc] initWithSiteId:self.siteID siteTimeZone:self.siteTimeZone oauth2Token:self.oauth2Token andCacheExpirationInterval:5 * 60 apiBaseUrlString:[WordPressComRestApi apiBaseURLString]];
```

Comment:
I haven't figured how we could inject the URL here 🤔

@ScoutHarris Do you know what scenario we could hit this code path and not have a valid `self.statsService` to pass through?

We could get away with leaving this as is, but I would prefer to cover all bases.